modules/govuk_env_sync/manifests/lock_file.pp

```diff
@@ -0,0 +1,15 @@
+# == Class: govuk_env_sync::lock_file
+#
+# Set permissions on the lock file
+#
+class govuk_env_sync::lock_file
+{
+
+  # Lock file
+  #
+  file { '/etc/unattended-reboot/no-reboot/govuk_env_sync':
+    mode  => '0660',
+    owner => $govuk_env_sync::user,
```

Comment:
You are using a variable that doesn't exist in this scope. It should be a param that is hiera'd or  default.